.github/workflows/pull_request_previews.yml

```diff
@@ -1,31 +1,33 @@
+# Synchronize pull requests to https://wptpr.live
+#
+# This task performs a potentially dangerous action (uploading code to
+# https://wptpr.live), so we restrict it to running on the base commit only
+# ('pull_request_target') so that malicious actors cannot modify the workflow
+# or scripts to force an upload.
```

Comment:
This is not 100% correct. The threat model here is write access to our GitHub **repo** (including code & settings), not to wptpr.live. wptpr.live by design does not have any sensitive information and the worst thing you could do is spamming (filling up its disk, or using it as free storage), which we guard against by checking author association (and banning abusive users).

Therefore, we are protecting the very powerful **DEPLOY_TOKEN** here *, which is accessible by this workflow (and any script it invokes). We don't want people modifying either and exposing the token, but we can allow members to push whatever they want (from branches or forks) to wptpr.live.

* See the original comment. This token actually has full `repo` access. And I believe it belongs to wpt-pr-bot, which is an admin, so technically you can even push directly to the master branch with this token...

Reply:
Thank you! That makes a lot more sense to me than what I originally thought the threat model was (I was quite confused as I wrote this comment, because the thread didn't make sense to me...).

Will get back to this comment once we resolve the conversation about `GITHUB_TOKEN` vs `DEPLOY_TOKEN` below...

Reply:
Yeah, just replace `DEPLOY_TOKEN` with `GITHUB_TOKEN` (and perhaps call out that this token has read/write access regardless of the origin of the PR).

Reply:
Done, PTAL at the new version.